content/unsupported-browser.html

```diff
@@ -4,15 +4,28 @@
 robots_tag: noindex
 ---
 
+<script defer src="/assets/js/unsupported-browser.js"></script>
+
 <section id="intro" class="intro-heading">
   <h1 class="mb-3">Unsupported Browser</h1>
   <p class="mb-1">
     Scratch Addons doesn't support the browser you're using.
   </p>
 </section>
 
-<section class="container">
-  <p>Use updated versions of Chrome, Firefox, or other browsers based on them to get started:</p>
+<div class="container">
+
+<section class="mb-5">
+  <p>Please update your browser to use Scratch Addons. We recommend exporting your settings before doing so.</p>
+  <div id="export-settings" style="display: none">
+    <button id="export-btn" class="btn btn-dark dark-btn-white">Export my extension settings</button>
+    <button id="export-view-btn" class="btn btn-dark dark-btn-white">View my extension settings</button>

```

Comment:
It doesn't really matter but I prefer shorter button labels.
```suggestion
    <button id="export-btn" class="btn btn-dark dark-btn-white">Export extension settings</button>
    <button id="export-view-btn" class="btn btn-dark dark-btn-white">View extension settings</button>
```

Reply:
I would try to keep the "my", as it makes it clear it's the user's settings, even though it might appear they currently are visiting a public website.

Reply:
There's already the mention of "exporting your settings" on the sentence before the buttons, and I don't see why would the user be confused without the "my".

Reply:
I don't see why the user would be confused with the "my" either, so let's keep it :P